gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -287,6 +287,43 @@ case class TransformPreOverrides(isAdaptiveContext: Boolean)
       throw new UnsupportedOperationException(s"${other.getClass.toString} is not supported.")
   }
 
+  private def processProjectExecTransformer(
+      project: ProjectExecTransformer): ProjectExecTransformer = {
+    // special treatment for Project containing all bloom filters:
```

Comment:
Maybe follow the comment style through by capitalizing the first letter and add `.` at the end.